spec/formatting.md

```diff
@@ -197,17 +197,31 @@ the following steps are taken:
 
 1. If the _expression_ includes an _operand_, resolve its value.
    If this fails, use a _fallback value_ for the _expression_.
-2. Based on the _function_ starting sigil and _name_,
-   find the appropriate function implementation from the _function registry_.
-   If the registry does not define an implementation for this _name_,
-   emit an Unknown Function error
+2. Resolve the _identifier_ of the _function_ and, based on the starting sigil,
+   find the appropriate function implementation to call.
+   If the implementation cannot find the function, emit an Unknown Function error
    and use a _fallback value_ for the _expression_.
-3. Resolve the _option_ values to a mapping of string identifiers to values.
+
+   An implementation MUST emit an Unknown Function error
+   if the implementation does not support _namespaces_
+   or if the _namespace_ of the _identifier_ is not recognized by the implementation
+   or the _namespace_ does not support the named _function_.
+
```

Comment:
I still don't see what this is adding to the preceding "If the implementation cannot find the function, emit an Unknown Function error". What sort of a situation would lead to that sentence not being sufficient to also cover everything added here?

Also, the unconditionality of its start now says that an implementation MUST "implement _namespaces_", or always emit an error at this point.

```suggestion
```

Reply:
it should say "if a namespace is present and the implementation does not support namespaces". Is that a sufficiently separate consideration from "can't find the function"?

Reply:
The question I'm trying to ask is, if a namespace is present and the implementation does not support namespaces, how can it find the corresponding function?

Reply:
It can't, which is they the MUSTard says "emit an error if you see a namespace and don't support namespaces"

Reply:
Now that we have 023b5a8, could this be removed?